server/schedule/opt/healthy.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opt
+
+import "github.com/pingcap/pd/server/core"
+
+// IsRegionHealthy checks if a region is healthy for scheduling. It requires the
+// region does not have any down or pending peers, and does not have any learner
+// peers when placement rules is disabled.
```

Comment:
I have a bit confused what means `does not have any learner peers when placement rules is disabled.`

Reply:
`And when placement rules feature is disabled, it requires the region does not have any learner peer`. Looks better?